integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -135,6 +136,11 @@ def resolve_secret(secret: Optional[Secret]) -> Optional[str]:
         self.stop_words = stop_words or []
         self.streaming_callback = streaming_callback
 
+        warnings.warn(
+            "The `meta` output of the AmazonBedrockChatGenerator will change in the next release to be inline with "
+            "OpenAI `meta`output keys.", stacklevel=2
```

Comment:
just a curiosity.
Could you briefly explain what `stacklevel=2` means?

Reply:
ask ruff :) he made me add it